### PR TITLE
Enh: Multi Input Style

### PR DESCRIPTION
**Pull request by mishingo**
**Is this a bugfix, or an enhancement? Please describe.**
Enhancement: improve appearance of multi input 

**Describe the solution you've implemented**
add style changes to update appearance

### CODE REVIEW

**Comment by mishingo** [comment]
image

**Comment by mishingo** [comment]
image

**Comment by qualman** [comment]
Looks good but looks like it is failing Selenium tests?

**Review by qualman — CHANGES_REQUESTED** [review]
Address failing Selenium test

**Comment by gschueler** [comment]
the borders around the Filter dropdown, and the Help button have a border-radius when it seems they shouldn't have one 
Screen Shot 2021-11-09 at 2 52 26 PM
Screen Shot 2021-11-09 at 2 52 32 PM

File: rundeckapp/grails-spa/packages/ui-trellis/src/components/inputs/MultiInput.stories.tsx [hunk 1]
```diff
@@ -11,66 +11,66 @@ export default {
...
-                    <i class ="glyphicon glyphicon-question-sign"></i>
-                    </a>
-                    <a class ="btn btn-primary btn-fill disabled" data-bind="click: $data.newFilterText, css: {disabled: !filter()}" href="#">
-                    Search
-                    </a>
-                    </div>
+                    </ul>
+                </div>
 
+                <script id="filter_select_tokens" type="text/json">{"TOKEN":"6cb1a494ae5310e236adf5662ef5047bd330075b8f7d74b7be906893ab755189","URI":"/project/zingbats-west-prod-8/nodes"}</script>
```
Comment (gschueler) [inline rundeckapp/grails-spa/packages/ui-trellis/src/components/inputs/MultiInput.stories.tsx:63]:
```suggestion
```

**Review by gschueler — CHANGES_REQUESTED** [review]

**Comment by qualman** [comment]
I cannot replicate the rounded corner issue and every other change looks good, so it's a go for me 👍🏻

**Review by qualman — APPROVED** [review]

**Review by gschueler — APPROVED** [review]

**Pull request merged**